### PR TITLE
Add Exercise05

### PR DESCRIPTION
**Pull request by huynhngocminh2511**

### CODE REVIEW

File: Exercise05/Adapters/MyAdapter.cs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+using Android.Widget;
+using Android.Support.V7.Widget;
+using System.Collections.Generic;
+using Exercise05.Models;
+using Android.Graphics;
+using Android.App;
+
+namespace Exercise05.Adapters
+{
+    class MyAdapter : RecyclerView.Adapter
```
Comment (voquanghoa) [inline Exercise05/Adapters/MyAdapter.cs]: Unacceptable class name.

File: Exercise05/Adapters/MyAdapter.cs [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    {
+        private List<Task> items;
```
Comment (voquanghoa) [inline Exercise05/Adapters/MyAdapter.cs]: Bad field name

File: Exercise05/Adapters/MyAdapter.cs [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+            var id = Resource.Layout.CardView;
+            var itemView = LayoutInflater.From(parent.Context).
+                   Inflate(id, parent, false);
+
+            return new MyAdapterViewHolder(itemView);
+        }
+
+        public override void OnBindViewHolder(RecyclerView.ViewHolder viewHolder, int position)
+        {
+            ((MyAdapterViewHolder)viewHolder).SetMyViewHolder(items[position]);
```
Comment (voquanghoa) [inline Exercise05/Adapters/MyAdapter.cs]: Wrong method name.

File: Exercise05/Adapters/MyAdapter.cs [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+
+            itemView.Click += delegate
+            {
+                var transcation = ((Activity)itemView.Context).FragmentManager.BeginTransaction();
+                var signup = new DialogClass($"Detail of {textViewProject.Text} {textViewExercise.Text}");
+                signup.Show(transcation, "Dialog Fragment");
+            };
+        }
+
+        public void SetMyViewHolder(Task task)
```
Comment (voquanghoa) [inline Exercise05/Adapters/MyAdapter.cs]: It should be `SetTask`

File: Exercise05/Adapters/MyAdapter.cs [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+        {
+            textViewProject.Text = task.Project;
+            textViewExercise.Text = task.Exercise;
+            textViewTopic.Text = task.Topic;
+            textViewDate.Text = task.Date.ToString("dd/MM/yyyy");
+            textViewTime.Text = task.Time.ToString();
+
+            if (task.DayLeft == 0)
```
Comment (voquanghoa) [inline Exercise05/Adapters/MyAdapter.cs]: Duplicate code. Please extract to a method with some useful parameters and reuse.

Comment (huynhngocminh2511) [reply Exercise05/Adapters/MyAdapter.cs]: Done

File: Exercise05/Adapters/MyAdapter.cs (hunk 4 above)
Comment (huynhngocminh2511) [reply Exercise05/Adapters/MyAdapter.cs]: Done

File: Exercise05/Adapters/MyAdapter.cs (hunk 3 above)
Comment (huynhngocminh2511) [reply Exercise05/Adapters/MyAdapter.cs]: Done

File: Exercise05/Adapters/MyAdapter.cs (hunk 2 above)
Comment (huynhngocminh2511) [reply Exercise05/Adapters/MyAdapter.cs]: Done

File: Exercise05/Adapters/MyAdapter.cs (hunk 1 above)
Comment (huynhngocminh2511) [reply Exercise05/Adapters/MyAdapter.cs]: Done

**Pull request merged**